src/components/FlowRunsBarChart.vue

```diff
@@ -161,6 +166,34 @@
 
     return buckets
   }
+
+  function getTimeRangeFromFlows(flowRuns: FlowRun[]): { expectedStartTimeAfter: Date, expectedStartTimeBefore: Date } {
```

Comment:
Is this just getting the min and max start times? This would be a lot simpler to use the `min` and `max`utilities from date-fns. 
```javascript
const maxDate = new Date()
const minDate = subSeconds(now, secondsInDay)
const startTimes = flowRuns.map(flowRun => flowRun.startTime)

return {
  expectedStartTimeAfter: min(minDate, ...startTimes),
  expectedStartTimeBefore: max(maxDate, ...startTimes)
}
```

Reply:
Lol, awe man, ty